Sources/SwiftKueryORM/Model.swift

```diff
@@ -837,6 +837,9 @@ public extension Model {
                     opr = operation.getOperator()
                     stringValue = operation.getStringValue()
                 } else if var array = value as? Array<Any> {
+                    if array.count < 1 {
```

Comment:
It's better to use `isEmpty`, because `count` has to actually count the elements in the array, which isn't necessary.